### PR TITLE
make galoshes obtainable and slowing you

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

shitters now can have noslips, but there will be debuffs

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- tweak: Galoshes are added to the Janidrobe.
- tweak: Galoshes now make you slower.

### CODE REVIEW

**Comment by IamVelcroboy** [comment]
This just punishes janis though.

**Comment by lzk228** [comment]
they can walk in ordinary shoes and wear galoshes only when mopping big puddles

**Comment by Hmeister-real** [comment]
I don't like this, pointless Jani nerf. Maybe make roundstart Galoshes different from vending galoshes if you have to, but I never really saw these as too big of an issue other than when nrp tiders or powergamers find dead or SSD janis.

>they can walk in ordinary shoes and wear galoshes only when mopping big puddles 

Sir, that is silly. Janis should not have to micromanage their shoes to optimize their cleaning speed or just to get rid of the passive slowdown, like sec does with sechuds and flash-immunity. Also janis without galoshes look weird, big yellow gloves mixed with basic shoes don't feel right

File: Resources/Prototypes/Entities/Clothing/Shoes/specific.yml [hunk 1]
```diff
@@ -90,6 +90,9 @@
   - type: Clothing
     sprite: Clothing/Shoes/Specific/galoshes.rsi
   - type: NoSlip
+  - type: ClothingSpeedModifier
+    walkModifier: 0.9
+    sprintModifier: 0.85
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Shoes/specific.yml:95]: 2 different mods is whacky.

**Comment by metalgearsloth** [comment]
What's the slowdown for them in 13

**Comment by IamVelcroboy** [comment]
Better idea: Budget galoshes

**Comment by metalgearsloth** [comment]
Okay I double-checked and as far as I can tell the magboots slowdown is 20% and galoshes is 10% so probably just use 0.9

File: Resources/Prototypes/Entities/Clothing/Shoes/specific.yml (hunk 1 above)
Comment (lzk228) [reply Resources/Prototypes/Entities/Clothing/Shoes/specific.yml:95]: fucked up

**Comment by KingOnBudget** [comment]
janibros.... its over...

**Comment by Hmeister-real** [comment]
o7, someone please revert with another solution.

God the big janicart is gonna be so shit to carry...

**Comment by K-Dynamic** [comment]
Why are we giving shitters galoshes lmao

**Comment by Golinth** [comment]
Making janitor gameplay even more of a slog of moving even slower because of shitters is not great way to solve this imo

**Comment by deltanedas** [comment]
give janitors acidifiers so shitters cant take their jordans

**Comment by LordEclipse** [comment]
> give janitors acidifiers so shitters cant take their jordans

Auto-detonate when a non-jani holds them

**Comment by Hanzdegloker** [comment]
I always drag the janitorial bucket, the slowdown from hauling that thing doesn't stack with the boots does it?

**Comment by Hanzdegloker** [comment]
bruh, all this means is breakins into the jani closet gonna go up by 200% since it's a guaranteed way to get no slips without having to fight a guy with no slips for em. Also that slowdown is gonna be PAIN.

**Comment by hord-brayden** [comment]
> Better idea: Budget galoshes

Or, toeless galoshes?

**Comment by IamVelcroboy** [comment]
> > Better idea: Budget galoshes
> 
> Or, toeless galoshes?

😄you might not slip but your socks will be wet and that's objectively worse!

**Comment by asperger-sind** [comment]
We've had galoshes in janivends for a while before they were removed, and they caused little-to-no (in my experience) problems, I don't see how them getting reintroduced (albeit nerfed) will cause any issues today.

**Comment by TurboTrackerss14** [comment]
coming back to comment on this to say "You were all right, powergamers did infact begin an uptick in janitor office getting broken into, and stealing noslips via hacking the vendor" 

They don't even put in the tiniest effort to walk up to hop and ask for the access.

**Comment by deltanedas** [comment]
this is why jordans were removed from vendor a while ago, people did exactly this

**Comment by Hanzdegloker** [comment]
I once had the HoP followed by the Cap come into my Jani Office to take the two janidrobe galoshes :/

**Comment by UbaserB** [comment]
i always had a bad feeling about this pr considering the reason why it was removed, and *now* people are starting to regret it lol.

**Comment by Djungelskog2** [comment]
> coming back to comment on this to say "You were all right, powergamers did infact begin an uptick in janitor office getting broken into, and stealing noslips via hacking the vendor"
> 
> They don't even put in the tiniest effort to walk up to hop and ask for the access.

yea but powergamings also not against the rules anymore :godo:

**Pull request merged**